modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/info/item/provider/test/ObjectEntryInfoItemFormProviderTest.java

```diff
@@ -509,9 +613,15 @@ private ServiceContext _updateServiceContext(
 	@Inject
 	private ListTypeEntryLocalService _listTypeEntryLocalService;
 
+	private ObjectDefinition _objectDefinitionAA;
+	private ObjectDefinition _objectDefinitionAA_AAA;
```

Comment:
rename `_objectDefinitionAA_AAA` to `_objectDefinitionAAA`.

tip: we use this pattern to `objectRelationship` like `_objectRelationshipAA_AAA`